packages/webapp/src/components/Forms/ManageCustomTypes/index.jsx

```diff
@@ -0,0 +1,89 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { PropTypes } from 'prop-types';
+import { AddLink } from '../../Typography';
+import PageTitle from '../../PageTitle/v2';
+import Form from '../../Form';
+import Tiles from '../../Tile/Tiles';
+import IconLabelTile from '../../Tile/IconLabelTile';
+import { tileTypes } from '../../Tile/constants';
+
+export default function ManageCustomTypes({
+  title,
+  handleGoBack,
+  addLinkText,
+  onAddType,
+  tileData,
+  formatTileData,
+  customTypeFieldName,
+  persistedFormData,
+  useHookFormPersist,
+  getIsSelected,
+  onTileClick,
+}) {
+  const { getValues, setValue, register, watch } = useForm({
+    defaultValues: persistedFormData,
+    mode: 'onChange',
+  });
+
+  useHookFormPersist(getValues);
+  register(customTypeFieldName);
+  const selectedTileId = watch(customTypeFieldName);
+
+  return (
+    <Form>
```

Comment:
Do we need this to be a form? 

Right now the manage custom types view only needs to be capable of single-click I think. So the click behaviour shouldn't need to be saved in form format.

Reply:
Definitely not. I didn't check what we had in the component for task type and just copied it...😅🙏